app/models/manageiq/providers/lenovo/physical_infra_manager/parsers/parser_dictionary_constants_2_0.rb

```diff
@@ -0,0 +1,9 @@
+module ManageIQ::Providers::Lenovo
+  # dictionary homologated for versions:
+  # 2.0
+  class ParserDictionaryConstants2X0 < ManageIQ::Providers::Lenovo::ParserDictionaryConstants
+    PROPERTIES_MAP = {
+      :led_identify_name => ["Identify", "Indetification"]
```

Comment:
You should use %(Identify Identification) instead of ["Identify", "Indetification"]. This statement will return a String Array.

It is really "Indetification"? Shouldn't be "Identification"?

Reply:
I Got it. My bad, it is really "Identification", thanks.

Reply:
No problem @douglasgabriel, just a typo! :smile: